server_config.cc

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include "server_config.h"
+#include <string>
+
+ServerConfig::ServerConfig(NginxConfig config) {
+	fillOutMap(config);
+}
+
+ServerConfig::~ServerConfig() {
+  // TODO
+}
+
+void ServerConfig::fillOutMap(NginxConfig config) {
+	// Initialize variables property, value
+	std::string property = "";
+	std::string value = "";
+	for (size_t i = 0; i < config.statements_.size(); i++) {
+		//search in child block
+		if (config.statements_[i]->child_block_ != nullptr) {
```

Comment:
Hmm, it still looks like the indentation in this block is still a little wonky and there seems to be a mix of tabs and spaces throughout the whole file. One thing you could try is to add 
```
"tab_size": 2,
"translate_tabs_to_spaces": true,
``` 
to the end of your sublime text user settings and that way you won't have to manually type out two spaces in the future!